searchhelper.user.js

```diff
@@ -15,7 +15,7 @@
 // @match      http://www.neopets.com/halloween/esophagor*.phtml
 // @match      http://www.neopets.com/faerieland/employ/employment.phtml*
 // @match      http://www.neopets.com/closet.phtml*
-// @match      http://www.neopets.com/auctions.phtml?*auction_id=*
+// @match      http://www.neopets.com/auctions.phtml?*
```

Comment:
Remove that question mark and I can pull! 👍🏽

Reply:
Oh missed that! Thanks